app/src/components/Common/EditEvent/EditEvent.tsx

```diff
@@ -17,28 +23,71 @@ interface IProps {
 export const EditEvent = ({ eventResult, updateEvent }: IProps) => {
   const event = eventResult;
   return (
-    <section className={commonStyle.content}>
-      <SectionWithValidation validationResult={event.title.errors}>
-        <section className={commonStyle.subsection}>
-          <TextInput
-            label={'title'}
-            placeholder="My event"
-            value={event.title.editValue}
-            onChange={title =>
-              updateEvent({
-                ...event,
-                title: validateTitle(title),
-              })
-            }
-          />
-        </section>
-      </SectionWithValidation>
-      <section
-        className={classNames(commonStyle.subsection, commonStyle.column)}
-      >
+    <>
+      <section className={style.editSection}>
+        <TextInput
+          label={'Tittel'}
+          placeholder=""
+          value={event.title.editValue}
+          onChange={title =>
+            updateEvent({
+              ...event,
+              title: validateTitle(title),
+            })
+          }
+        />
+        <ValidationResult validationResult={event.title.errors} />
+      </section>
+      <div className={commonStyle.row}>
+        <div className={style.colHalf}>
+          <section className={style.editSection}>
```

Comment:
Jeg synes det er litt rart at komponenten består av `section`s, unntatt denne biten som er to `section`s wrappa i `div`s. Det er vel mer riktig semantisk å ha en `section` wrappa rundt `div`ene her?